contracts/colony/Colony.sol

```diff
@@ -20,11 +20,12 @@ pragma experimental ABIEncoderV2;
 
 import "./../common/ERC20Extended.sol";
 import "./../common/IEtherRouter.sol";
+import "./../common/GetChainId.sol";
```

Comment:
Can we call this `MultiChain`? It doesn't seem right to name a contract after a function.